numba/core/ir_utils.py

```diff
@@ -1303,7 +1303,7 @@ def find_single_branch(label):
 
 
 arr_math = ['min', 'max', 'sum', 'prod', 'mean', 'var', 'std',
-            'cumsum', 'cumprod', 'argmin', 'argmax', 'argsort',
+            'cumsum', 'cumprod', 'argmin', 'argsort',
```

Comment:
This change is probably what's causing parfors (parallelisation) tests to fail. By removing this string expressions like `<array instance>.argmax()` can't be canonicalised to `np.argmax(<array instance>)` and as a result they don't get spotted by the parallel function replacement pass. Is this change needed to make the rest of the tests pass?

Reply:
Thanks for the quick response!

And yes, I was over-aggressive apparently detaching the `lower_builtin` code and shouldn't have touched this bit.